pom.xml

```diff
@@ -128,19 +128,19 @@
     <zookeeper.version>3.4.14</zookeeper.version>
     <curator.version>2.7.1</curator.version>
     <okapi.version>0.4.2</okapi.version>
-    <hive.group>org.spark-project.hive</hive.group>
-    <hive.classifier></hive.classifier>
+    <hive.group>org.apache.hive</hive.group>
+    <hive.classifier>core</hive.classifier>
     <!-- Version used in Maven Hive dependency -->
-    <hive.version>1.2.1.spark2</hive.version>
+    <hive.version>2.3.6</hive.version>
     <hive23.version>2.3.6</hive23.version>
     <!-- Version used for internal directory structure -->
-    <hive.version.short>1.2.1</hive.version.short>
+    <hive.version.short>2.3.5</hive.version.short>
```

Comment:
BTW, any reason why we needed the patch version in the directory names initially? I don't think we need different shims for Hive patch versions. If that's true, I'd suggest renaming the `sql/hive-thriftserver/{v1.2.1,v2.3.5}` folders to just `v1.2` and `v2.3`. Of course, this can be done in a follow-up PR.

Reply:
Sure. We can use the short version. I'll do in the next follow-up PR. It will include renaming files mostly.

Reply:
Since we are already in Apache Hive 2.3.6, I don't expect many changes at 2.3.7+.

Reply:
@dongjoon-hyun i found hive storage api and hive common have the following common class files
```
org/apache/hadoop/hive/common/ValidReadTxnList.class
org/apache/hadoop/hive/common/ValidTxnList.class
org/apache/hadoop/hive/common/ValidTxnList$RangeResponse.class
```
For example, https://github.com/apache/hive/blob/rel/storage-release-2.6.0/storage-api/src/java/org/apache/hadoop/hive/common/ValidReadTxnList.java (pulled in by orc 1.5.8) and https://github.com/apache/hive/blob/rel/release-2.3.6/common/src/java/org/apache/hadoop/hive/common/ValidReadTxnList.java (from hive-common 2.3.6) both are in the classpath and they are different. I am worried that users may hit issues due to classloading order. I think it will be safe to still use nohive classifier for orc, which shades storage api.

Reply:
I think there is no downside of using orc nohive, right? Why not just keep using that. So, we will not need to worry about having class conflicts.

Reply:
sorry. I should put these comments in the comments of this pr not here.I will move them.